### PR TITLE
FEA-81-Add button favorites to single page

### PR DESCRIPTION
**Pull request by AgataLudwiczynska-CC**

### CODE REVIEW

File: src/components/FavoritesButton/FavoritesButton.tsx [hunk 1]
```diff
@@ -21,7 +22,7 @@ const FavoritesButton = ({ cityName }: Props) => {
 
   return (
     <Button data-testid={FavoritesButtonTestIds.Button} active={isFavorite} onClick={handleClick}>
-      <Heart size={13} />
+      <Heart size={size} />
```
Comment (klaudia-blazyczek-cc) [inline src/components/FavoritesButton/FavoritesButton.tsx:25]: **Suggestion** What do you think about adding the size to some kind of theme?

**Review by kamil-borecki-cc — APPROVED** [review]

**Review by darya-karpovich-cc — COMMENTED** [review]
Maybe you should add test to city page is the button rendered

File: src/components/FavoritesButton/FavoritesButton.tsx (hunk 1 above)
Comment (AgataLudwiczynska-CC) [reply src/components/FavoritesButton/FavoritesButton.tsx:25]: I don't know if this is a good solution, because this value is different for each view

Comment (klaudia-blazyczek-cc) [reply src/components/FavoritesButton/FavoritesButton.tsx:25]: Okay

**Review by darya-karpovich-cc — APPROVED** [review]

**Review by klaudia-blazyczek-cc — APPROVED** [review]

**Pull request merged**